sdk/python/tests/facade/test_SymbolFacade.py

```diff
@@ -345,6 +345,23 @@ def test_can_cosign_transaction_with_account_wrappers(self):
 	def test_can_cosign_transaction_detached_with_account_wrappers(self):
 		self._run_test_can_cosign_transaction_detached(self._get_cosign_transaction_test_descriptor_for_account_wrappers())
 
+	@staticmethod
```

Comment:
this can probably be defined within scope of _get_cosign_transaction_hash_test_descriptor_for_account_wrappers (only called once).

same comment for function pair below too.